### PR TITLE
[css-nav-1] Add spatial-navigation-action CSS Property

### PR DESCRIPTION
**Pull request by jihyerish**
Add the `spatial-navigation-action` CSS property which can specify the spatial navigation behavior of the container.

IDL:
```css
spatial-navigation-action: auto | focus | scroll
```
* `auto`: The directional input moves the focus to the visible candidates inside the spatial navigation container. If there isn't any visible candidate, it scrolls the container.
* `focus`: The directional input moves the focus to candidates inside the spatial navigation container. The user cannot manually scroll the container.
* `scroll`: The directional input scrolls the container manually.
(`scroll` value is noted as 'at-risk')

Resolves: https://github.com/w3c/csswg-drafts/issues/3401

### CODE REVIEW

File: css-nav-1/Overview.bs [hunk 1]
```diff
@@ -1392,6 +1250,79 @@ Note: the 'spatial-navigation-contain' property is <a>at-risk</a>.
...
 </div>
 
+<h3 id=css-property-spatialnavigationaction>
+Defining the spatial navigation behavior : the 'spatial-navigation-action' property</h3>
+
+<pre class='propdef'>
+Name: spatial-navigation-action
+Value: auto | focus | scroll
+Initial: auto
+Applies to: <a>spatial navigation focus container</a>
```
Comment (jihyerish) [inline css-nav-1/Overview.bs]: @frivoal Which is the most correct description among:
(1) all element
(2) spatial navigation focus container
(3) scrollable spatial navigation focus container

(1) is possible, but specifying this property is meaningless for the element which isn't a spatial navigation container.

Comment (anawhj) [reply css-nav-1/Overview.bs]: You can refer to the following links in terms of the consideration above.
https://www.w3.org/TR/cssom-view-1/#smooth-scrolling
https://www.w3.org/TR/css-scroll-snap-1/#scroll-padding

Comment (jihyerish) [reply css-nav-1/Overview.bs]: I think this property is meaningful for the scrollable element.
Other properties such as https://drafts.csswg.org/cssom-view/#smooth-scrolling uses 
`scrolling box` as its applied element, so I'll change to it.

Comment (anawhj) [reply css-nav-1/Overview.bs]: Great! seems no problem now. +1

**Pull request merged**